pkg/kubernetes/scripts/object-describer.js

```diff
@@ -431,7 +431,7 @@ angular.module('kubernetesUI').run(['$templateCache', function($templateCache) {
     "    <dd>\n" +
     "      <div ng-if=\"!resource.spec.ports.length\">None</div>\n" +
     "      <div ng-repeat=\"portMapping in resource.spec.ports | orderBy:'port'\">\n" +
-    "        {{portMapping.port}} &#8594; {{portMapping.targetPort}} ({{portMapping.protocol}})\n" +
+    "        {{portMapping.port}} &#8594; {{portMapping.targetPort}} ({{portMapping.protocol}}) {{portMapping.nodePort?'NodePort: '+portMapping.nodePort:''}}\n" +
```

Comment:
I would suggest using a tag like this: ```<span ng-if='portMapping.nodePort' translatable="yes">Node Port:</span>``` since it allows for translation and doesn't mix labels with the values.

The object-describer comes from another repository. Could you also submit a pull request upstream? 

https://github.com/kubernetes-ui/object-describer